react/features/conference/route.js

```diff
@@ -85,9 +85,6 @@ function _obtainConfigAndInit() {
                         descriptionKey: 'dialog.connectError'
                     });
                 });
-        } else {
-            chooseBOSHAddress(config, room);
```

Comment:
Why is this not needed anymore? Where is the conference initialized then?

Reply:
This is some experiment that Boris was doing before adding the parameter to the bosh url to be used for the shard selection. Or maybe I deleted too much :) I wanted to remove the boshlist part. I will revisit it now.

Reply:
The removed part is handling only config.boshList, which was never used.

Reply:
Buuut _initConference() was not supposed to be removed. :) Reverting this part now.